scalableminds/webknossos
# Also load up to three fallback layers for 'best quality first'

Right now, the rendering strategies define how many fallback magnifications will be loaded so that these can be rendered. For progressive loading, three fallback magnifications are loaded. For best quality, only one fallback mag was loaded until now. The reasoning behind this was that additional fallbacks shouldn't be necessary (since they are loaded with lower priority). However, this had the effect that missing data couldn't be covered by fallback data if that fallback was more than one mag apart.
I simply changed the value of how many fallback mags should be loaded to three. This can produce more traffic, but since the lower mags have lower priority when using "best quality first", this shouldn't affect the perceived loading speed.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset for which data only exists in mag 4 or mag 8.
- Select "best quality first" and disable "render missing data black"
- Data should be visible when zooming to mag 1

### Issues:
- fixes #4444 

------
